customerio/client_base.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import division
```

Comment:
Now that we've client_base.py and track.py, I think we should add a one-line comment at the top of these files to explain what these are.